collectors/ebpf.plugin/ebpf.conf

```diff
@@ -1,18 +1,44 @@
+#
+# Global options
+#
+# The `ebpf load mode` option accepts the following values :
+#  `entry` : The eBPF collector only monitors calls for the functions, and does not show charts related to errors.
+#  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
+#            new charts for the return of these functions, such as errors.
+#
+# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`.
+# If you want to disable the integration with `apps.plugin` along with the above charts, change the setting `apps` to
+# 'no'.
+#
 [global]
     ebpf load mode = entry
-    disable apps = no
+    apps = yes
 
+#
+# eBPF Programs
+#
+# The eBPF collector enables and runs the following eBPF programs by default:
+#
+#  `process`       : This eBPF program creates charts that show information about process creation, VFS IO, and
+#                    files removed.
+#  `network viewer`: This eBPF program creates charts with information about `TCP` and `UDP` functions, including the
+#                    bandwidth consumed by each.
 [ebpf programs]
     process = yes
     network viewer = yes
-    network connection monitoring = no
+    network connection = no
```

Comment:
I think plural form is more appropriate here: "network connection" -> "network connections".